ipv-core/src/main/java/com/ibm/whc/deid/providers/masking/AbstractMaskingProvider.java

```diff
@@ -69,6 +69,10 @@ public AbstractMaskingProvider(String tenantId, String localizationProperty,
   public String mask(String identifier, String fieldName) {
     return mask(identifier);
   }
+  
+  public String maskConstant(String identifier) {

```

Comment:
I would not add this method.  The rest of the system won't know about it or call it.  All of our testing should be via the methods that are already in the MaskingProvider interface, as those are the methods the real system will call.  In this case, **public String mask(String identifier)**.  Just incorporate the new checks into that method.